erizo/src/erizo/rtp/QualityManager.cpp

```diff
@@ -59,6 +59,8 @@ void QualityManager::onConnectionQualityUpdate(ConnectionQualityLevel level) {
     selectLayer(false);
   }
   last_connection_quality_level_received_ = level;
+  stats_->getNode()["qualityLayers"].insertStat("currentQualityLevel",
```

Comment:
I've decided to add the stat here in `QualityManager`. This is where the quality check changes affect the decisions. I could be convinced to put it directly in `BandwidthConnectionQualityCheck`, though.